Sources/Bot/Services/MergeService.swift

```diff
@@ -353,28 +353,43 @@ extension MergeService {
     fileprivate static func whenIntegrationFailed(
         github: GitHubAPIProtocol,
         logger: LoggerProtocol,
-        scheduler: Scheduler
+        scheduler: Scheduler,
+        extendedLogging: Bool
     ) -> Feedback<State, Event> {
 
         struct IntegrationHandler: Equatable {
             let pullRequest: PullRequest
             let integrationLabel: PullRequest.Label
             let failureReason: FailureReason
+            let trace: Trace
+            let extendedLogging: Bool
 
             var failureMessage: String {
-                return "@\(pullRequest.author.login) unfortunately the integration failed with code: `\(failureReason)`."
+                let message = "@\(pullRequest.author.login) unfortunately the integration failed with code: `\(failureReason)`."
+                if extendedLogging {
+                    return message + "\n<details>function: \(trace.function) line: \(trace.line)</details>"
```

Comment:
Feels it would be more natural to make `trace` be `CustomStringConvertible` and just use `<details>\(trace)</details>` here

Reply:
Maybe.